plugins/outputs/sematext/processors/heartbeat.go

```diff
@@ -0,0 +1,71 @@
+package processors
+
+import (
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+	"time"
+)
+
+const (
+	oneMinuteSeconds int64 = 60
+)
+
+// Heartbeat is a batch processor that injects heartbeat metric as necessary (once per minute).
+type Heartbeat struct {
+	lastInjectedMinute int64
```

Comment:
can this variable be accessed by multiple goroutines? If so, we should make it thread-safe.

Reply:
Good catch - from what I saw, that shouldn't be possible, but I will adjust this variable so we are safe.